### PR TITLE
fix: [BG-54]: fix e.split is not a function

### PR DESCRIPTION
**Pull request by Sagar-Rajput7**
##### Jira Links:

https://harness.atlassian.net/browse/BG-54

##### Screenshots:

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by Sagar-Rajput7** [comment]
trigger cypress

**Review by vkbansal — APPROVED** [review]

File: src/modules/70-pipeline/components/LogsContent/LogsState/updateSectionData.ts [hunk 1]
```diff
@@ -12,25 +12,27 @@ import { sanitizeHTML } from '@common/utils/StringUtils'
...
-        accumulator.push({
-          text: {
-            level: sanitizeHTML(level),
-            time: formatDatetoLocale(time),
-            out: sanitizeHTML(out)
-          }
-        })
-      } catch (e) {
-        //
+  return String(data)
```
Comment (chetanvekariya-harness) [inline src/modules/70-pipeline/components/LogsContent/LogsState/updateSectionData.ts:15]: Nothing wrong with this change, but It should work even without this change. Issue was not here, but in **FullPageLogView.tsx** component which you have fixed below.

**Review by chetanvekariya-harness — APPROVED** [review]

File: src/modules/70-pipeline/components/LogsContent/LogsState/updateSectionData.ts (hunk 1 above)
Comment (Sagar-Rajput7) [reply src/modules/70-pipeline/components/LogsContent/LogsState/updateSectionData.ts:15]: Ok, it's just Vivek told me to keep both changes.
So, if there is no such major issue here, @chetanvekariya-harness  can I merge this?
Thanks

Comment (vkbansal) [reply src/modules/70-pipeline/components/LogsContent/LogsState/updateSectionData.ts:15]: The error was `split is not a fn` and not `cannot read property split of undefined`. So the assumption here is that we are getting a non-string value. Hence, asked him to typecast it.

**Pull request merged**